src/style.css

```diff
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
```

Comment:
You have to add here the rule `font-weight: 700;` for "Heavy" font

Reply:
done